algorithm/cgal/wrap.js

```diff
@@ -0,0 +1,22 @@
+import { STATUS_OK, STATUS_ZERO_THICKNESS } from './status.js';
+import { fromCgalGeometry, withCgalGeometry } from './cgalGeometry.js';
+
+import { ErrorZeroThickness } from './error.js';
+
+export const wrap = (inputs, alpha, offset) =>
```

Comment:
What is wrap doing? Wrapping a set of points?

Reply:
It wraps a volume around a set of points and/or faces, but it doesn't work very well yet.